pkg/asset/agent/image/agentpxefiles.go

```diff
@@ -179,3 +194,26 @@ boot
 
 	return nil
 }
+
+func getKernelArgs(isoPath, file string) (string, error) {
+	fileReader, err := isoeditor.GetFileFromISO(isoPath, file)
```

Comment:
Couldn't be retrieved also this one from the tmp path?

Reply:
Done